packages/block-editor/src/components/block-list/block.js

```diff
@@ -269,6 +270,13 @@ function BlockListBlock( {
 			toggleSelection={ toggleSelection }
 		/>
 	);
+
+	// For aligned blocks, provide a wrapper element so the block can be
+	// positioned relative to the block column.
+	if ( attributes.align ) {
```

Comment:
I'm wondering if this is a good way to check whether a block has alignment. For me a more precise way is to check the wrapper props and see if there's a "data-align" because that's how floating is triggered.

Reply:
Sounds good. I changed it to `data-align`.